imports/ui/transactions/ListContainer.js

```diff
@@ -9,7 +9,7 @@ export default ValidatorDetailsContainer = withTracker((props) => {
 
     if (Meteor.isClient){
         transactionsHandle = Meteor.subscribe('transactions.list', props.limit);
-        loading = !transactionsHandle.ready();
+        loading = !transactionsHandle.ready() && props.limit == Meteor.settings.public.initialPageSize;
```

Comment:
@MonikaCat yes this would fix the display issue. However, if the txs are being loaded continuously, the DOM will be rendering too many components if the users keeps scrolling. Can you do a test and see if the screen will be laggy after some scrolls?

Reply:
Sure! The screen becomes a little bit laggy after scrolling down multiple times. 

I have wrapped the list in Container so we can track scrolling only of the list component. The overall performance of the page has improved and the transactions are loading faster. What do you think?

Reply:
I don't understand. Why wrapped inside a `<Container>` would improve performance? I think the `<List>` component is still querying and rendering components controlled by `limit`?